web-ui/src/app.d.js

```diff
@@ -0,0 +1,28 @@
+// Type definitions specified here will be availalbe app-wide
+
+/**
+ * @typedef {Object} Member
+ * @property {?string} bioText
+ * @property {?Array} birthday
+ * @property {string} empolyeeId
```

Comment:
`employeeId`?

Reply:
The type name has been changed from `Member` to `MemberProfile` to better match the routes we call on the server for this data, in this case `/member-profiles` endpoint. The endpoint returns `employeeId` like:

<img width="576" alt="Screenshot 2024-05-07 at 10 12 55 AM" src="https://github.com/objectcomputing/check-ins/assets/97140109/87723fc0-1dc8-47f3-84fa-6b566551e642">

Reply:
@vhscom I'm sorry. I should have been more clear. I was pointing out that it was misspelled.

Reply:
Good eyes. Thank you.